README.md

```diff
@@ -277,62 +287,137 @@ signIn.renderEl(
     // screen letting them know that an email is on the way.
     if (res.status === 'FORGOT_PASSWORD_EMAIL_SENT') {
       // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
       return;
     }
 
     // The user has started the unlock account flow, and is on the confirmation
     // screen letting them know that an email is on the way.
     if (res.status === 'UNLOCK_ACCOUNT_EMAIL_SENT') {
       // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
+      return;
+    }
+
+    // The user has started the activation flow, and is on the confirmation
+    // screen letting them know that an email is on the way.
+    if (res.status === 'ACTIVATION_EMAIL_SENT') {
+      // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
+      return;
+    }
+
+    // The user has completed the registration flow
+    if (res.status === 'REGISTRATION_COMPLETE') {
+      // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - activationToken (string)
       return;
     }
 
     // The user has successfully completed the authentication flow
     if (res.status === 'SUCCESS') {
 
-      // Handle success when the widget is not configured for OIDC
-
       if (res.type === 'SESSION_STEP_UP') {
         // Session step up response
         // If the widget is not configured for OIDC and the authentication type is SESSION_STEP_UP,
         // the response will contain user metadata and a stepUp object with the url of the resource
         // and a 'finish' function to navigate to that url
+        // Properties available on `res`:
+        // - status (string)
+        // - user (object)
+        // - type (string)
+        // - stepUp (object)
+        //   - url (string)
+        //   - finish (function)
         console.log(res.user);
         console.log('Target resource url: ' + res.stepUp.url);
         res.stepUp.finish();
         return;
-      } else {
-        // If the widget is not configured for OIDC, the response will contain
-        // user metadata and a sessionToken that can be converted to an Okta
-        // session cookie:
+      }
+
+      if (res.next) {
```

Comment:
One of the confusing parts in the documentation before was that users did not have any understanding of when `next()` would be available and when `session.setCookieAndRedirect` would be available. I'm trying to differentiate here by checking for `next` as a property. Would love feedback here and on the documentation I've included for this block.

Reply:
`setCookieAndRedirect` is available at the end of the flow when the widget is not configured for OIDC. It is used by customers in some federated / IDP scenarios. Use of this function has been identified as an "anti-pattern" in most cases because it uses an internal object (sessionToken) and allows an arbitrary redirect. It will be great to document how it works, but we should also try to nudge users towards using standard OIDC flows